arangodb-net-standard/CollectionApi/GetCollectionResponse.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Net;
+
+namespace ArangoDBNetStandard.CollectionApi
+{
+    public class GetCollectionResponse
```

Comment:
In the project, we have model names that ends with `Response` and others with `Result`. Should we decide on one naming convention?

I have a preference for `Result` because it's the "result" of an operation and not the "response" but rather it's content.

Reply:
Well response and request are key words I word associate with HTTP. And our client is returning the result. So yes I would agree with using Result instead. Make an issue for it?

Reply:
Following discussion we had elsewhere, we are sticking with `Response` as a standard suffix.  The logic is that what we give back is the application response from ArangoDB.